### ISSUE TITLE
Sortable setting doesn't work on load if used on first column

### ISSUE THREAD

**Issue by erasaur** [aslagle/reactive-table#189]
If you set `sortable: false` on the first column, when the table initially loads the column is still sortable.

**Comment by aslagle** [comment]
You're right, I think I've got a related issue somewhere. There is a workaround - if you set `sort` on one of the other columns, it will sort by that column initially instead of using the first column as the default. There's no way to disable sorting on all the columns.

**Comment by erasaur** [comment]
Oh, I see. The workaround will do for now.
